npm_and_yarn/lib/dependabot/npm_and_yarn/file_parser/lockfile_parser.rb

```diff
@@ -52,6 +53,24 @@ def potential_lockfiles_for_manifest(manifest_filename)
             compact
         end
 
+        def instrument_package_manager_version
+          package_managers = {}
+
+          package_locks.each do |lockfile|
+            versions = [package_managers["npm"]]
+            versions << Helpers.npm_version_numeric(lockfile) 
```

Comment:
This should fix that error

```suggestion
            versions << Helpers.npm_version_numeric(lockfile.content) 
```